packages/next/package.json

```diff
@@ -314,7 +312,8 @@
     "webpack": "5.86.0",
     "webpack-sources1": "npm:webpack-sources@1.4.3",
     "webpack-sources3": "npm:webpack-sources@3.2.3",
-    "ws": "8.2.3"
+    "ws": "8.2.3",
+    "zod": "3.22.2"
```

Comment:
can we upgraed zod to `3.22.3` due to the security issue https://security.snyk.io/vuln/SNYK-JS-ZOD-5925617

Reply:
@huozhi Sure! I will update the zod after bringing up zod error message normalization.

Reply:
Done in [`af5bb95` (#56383)](https://github.com/vercel/next.js/pull/56383/commits/af5bb95e31070d7daf4c53c8109973b26880fc73)